content/reference/v1beta1/config.md

```diff
@@ -235,9 +239,23 @@ Items can be readonly.
           readonly: true
 ```
 
```

Comment:
My understanding is it’s more than just random string

Reply:
@emosbaugh Oh! So does that mean _any_ function that generate "things" (RandomString being just one of them) will regenerate based on `hidden` vs `readonly`?

Reply:
I think that's right @aj-jester but I'll let @emosbaugh confirm

Reply:
Updated the doc to be more generic.